.baseline/checkstyle/checkstyle.xml

```diff
@@ -189,6 +189,16 @@
             <property name="illegalPkgs" value="com.google.common"/>
             <message key="import.illegal" value="Use org.apache.iceberg.relocated.* classes from bundled-guava module instead."/>
         </module>
+        <module name="IllegalImport">
+            <property name="id" value="BanUnrelocatedGuavaClasses"/>
```

Comment:
Nit: The value here looks like i was copy pasted. Should maybe be `BanUnrelocatedAssertjUtilClasses`.

Reply:
I was mostly following this:

```
<module name="IllegalImport">
            <property name="id" value="BanUnrelocatedGuavaClasses"/>
            <property name="illegalPkgs" value="com.google.common"/>
            <message key="import.illegal" value="Use org.apache.iceberg.relocated.* classes from bundled-guava module instead."/>
        </module>
```

I am actually not sure if the ID has to be unique, let me check

Reply:
Ah I see, so that's a user-defined name, then let me update it

Reply:
I checked myself when I was verifying this (not 100% used to updating these files), and they do not have to be unique.

It's just used in the output it seems, like this example where I added an import.

```
> Task :iceberg-arrow:checkstyleMain FAILED
[ant:checkstyle] [ERROR] /Users/kylebendickson/repos/iceberg-kyle/arrow/src/main/java/org/apache/iceberg/arrow/ArrowAllocation.java:23:1: Use org.apache.iceberg.relocated.* classes from bundled-guava module instead. [BanUnrelocatedGuavaClasses]
```